pkg/event-consumers/kafka/kafka-consumer.go

```diff
@@ -84,6 +85,12 @@ func createConsumerProcess(broker, topic, funcName, ns, consumerGroupID string,
 	}
 }
 
+func isJSON(s string) bool {
```

Comment:
Where is the unittest?
This function only works for JSON *objects*.  (That may or may not be ok?)

Reply:
let me submit a different PR for covering this file with unit tests